alphastats/gui/pages/06_LLM.py

```diff
@@ -130,6 +133,60 @@ def llm_config():
             unsafe_allow_html=True,
         )
 
+st.markdown("##### Select which information from Uniprot to supply to the LLM")
```

Comment:
The options and preview are each in an expander and the options are collapsed by default. Fully collapsed this would still show the row of buttons and the two expanders.

Reply:
I would not want to turn the whole thing into an expander, as I can't nest them and I would prefer having the button visible at all times. This might change however, when it's actually passed to the LLM.